sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -812,8 +812,8 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
-  val AUTO_UPDATE_SIZE =
-    buildConf("spark.sql.statistics.autoUpdate.size")
+  val AUTO_UPDATE_ENABLED =
```

Comment:
How about `AUTO_UPDATE_SIZE_ENABLED`? If we support auto update other stats (needs table scan in addition to just looking up file length) in the future, we may need another config.

Reply:
`AUTO_SIZE_UPDATE_ENABLED`